buildspecs/merge-build.yml

```diff
@@ -0,0 +1,55 @@
+version: 0.2
+
+env:
+  git-credential-helper: yes
+  exported-variables:
+    - CODEBUILD_BUILD_ID
+    - ECS_AGENT_TAR
+
+phases:
+  install:
+    commands:
+      - architecture=""
+      # Same buildspec for different architectures - detect the architecture here and rename the artifacts accordingly
+      - |
+        case $(uname -m) in
+          x86_64)
+            architecture="amd64"
+          ;;
+          aarch64)
+            architecture="arm64"
+          ;;
+        esac
+
+      - GOVERSION="$(cat GO_VERSION)"
+      - GOLANG_TAR="go${GOVERSION}.linux-${architecture}.tar.gz"
+
+      # Need to install GOLANG explicitly as required versions do not come preinstalled
+      - wget -O /tmp/${GOLANG_TAR} https://storage.googleapis.com/golang/${GOLANG_TAR}
+      - tar -C /usr/local -xzf /tmp/${GOLANG_TAR}
+      - go version
+
+      # glibc-static required for dockerfree make method
+      - echo "Install glibc-static for dockerfree make method"
+      - yum -y install glibc-static
```

Comment:
not a blocker but https://github.com/aws/amazon-ecs-agent/pull/3206 we don't need this if we use pre-built pause containers.

Reply:
👍